scripka/Overlook-project
# Adding styles for login page and sample images

## What is the change?
Added new elements to html for  a manager view.
Added styles for the login page

## What does it fix?
N/A

## Is this a fix or a feature? 
Feature

## Where should the reviewer start? 
styles.scss: line 1
index.html: line 4-7 adding links, line 14-71 - adding new chart element for displaying data.

## How should this be tested?
When running npm start if the hidden class is removed from the login element you can see the login window that users can use to enter username and password.
